source/config/profileUpgradeSteps.py

```diff
@@ -118,3 +118,19 @@ def upgradeConfigFrom_6_to_7(profile: Dict[str, str]) -> None:
 		selectiveEventRegistration = False
 	if selectiveEventRegistration:
 		profile['UIA']['eventRegistration'] = "selective"
+
+
+def upgradeConfigFrom_7_to_8(profile: Dict[str, str]) -> None:
+	"""
+	In Document formatting settings, "Row/column headers" check box has been replaced with "Headers" combobox.
+	"""
+	try:
+		reportTableHeaders = profile['documentFormatting']['reportTableHeaders']
+	except KeyError:
+		# Setting does not exist, no need for upgrade of this setting
+		log.debug("reportTableHeaders not present, no action taken.")
+	else:
+		if reportTableHeaders:
+			profile['documentFormatting']['reportTableHeaders'] = 1  # Rows and columns

```

Comment:
Can't you import the enum with possible values here and use `member.value` rather than an integer literal and a comment explaining what the value means?

Reply:
Done in 450cfec36e9fca0c58713c7ce35ac32565296beb.